Jenkinsfile

```diff
@@ -84,3 +83,32 @@ demos['databound'] = {
 }
 
 parallel(demos)
+
+def branchName = currentBuild.projectName
+if (!branchName.startsWith('PR-')) {
+    node('docker') {
+        def image
+        def imageName = 'jenkins/jenkinsfile-runner-experimental'
```

Comment:
It will not work ad you may expect. Experimental images must go to another DockerHub organization. 
See the naming recommendations in https://github.com/jenkinsci/jep/tree/master/jep/217#implementation-on-cijenkinsio

Reply:
Do you mean  
`def imageName = '${env.DOCKERHUB_ORGANISATION}/jenkinsfile-runner-experimental'`
?

Reply:
Yes

Reply:
ok!. Thanks for the clarification. I didn't understand correctly that excerpt then.